BE/toDoListAPI/src/main/java/shcool/mindera/toDoListAPI/repository/UserRepository.java

```diff
@@ -0,0 +1,7 @@
+package shcool.mindera.toDoListAPI.repository;
+
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class UserRepository {
```

Comment:
Mete já o extends JPARepo e transforma para interface 
```suggestion
public interface UserRepository  extends  JpaRepository<T,T>{
```

Reply:
Mano nao te esquecas que isto é um ticket de entidades, isso é so para teste

Reply:
Mas se já está uma class para teste , porque não deixar a interface que vai estar